### ISSUE TITLE
Broken sidebar navigation

### ISSUE THREAD

**Issue by LukasHirt** [owncloud/owncloud-design-system#812]
For a long time, there is broken navigation in the sidebar. To display sub-items we need to either click twice on the parent item or we need to refresh the page. This is caused because the hiding/displaying of sub-items is handled via custom logic but the way how the docs are rendered has changed. We can try to investigate time to fixing this custom logic or we can use the default behaviour of styleguidist. Upstream has an opened ticket about this but it seems that the project is not active at all anymore.

**Comment by kulmann** [comment]
yes, this was bugging me as well... important to have working documentation for both community and ourselves => we should schedule time for this 👍

**Comment by rpocklin** [comment]
I found this today and was going to raise it as a new issue, found it already exists.

Would be great to fix, I thought I was doing something wrong for a bit.

Here is a video (watch until the end for the clearest example)
I would say there is some bad event handling or double-event binding going on (or perhaps a timing issue).

[owncloud-design-com-sidebar-click-event-issue.zip](https://github.com/owncloud/owncloud-design-system/files/5273427/owncloud-design-com-sidebar-click-event-issue.zip)

**Comment by LukasHirt** [comment]
> I would say there is some bad event handling or double-event binding going on (or perhaps a timing issue)

Yep, I was once trying to debug why it's happening and found out there is some re-render taking place after the custom logic for displaying sub menus. This started happening after an update of vue-styleguidist which handles navigation differently than us.
